resources/windows/wix/license/LICENSE.rtf

```diff
@@ -0,0 +1,375 @@
+{\rtf1\ansi\ansicpg1252\deff0{\fonttbl{\f0\fswiss\fcharset0 ArialMT;}{\f1\fnil\fcharset0 Courier New;}{\f2\fnil\fcharset0 Arial;}}
+{\colortbl ;\red0\green0\blue255;\red57\green57\blue57;}
+{\*\generator Msftedit 5.41.21.2510;}\viewkind4\uc1\pard\lang9\b\f0\fs28 License Summary:\b0\fs20\par
+\par
+\pard\fi-360\li360 Puppet - Apache 2.0\par
+Facter - Apache 2.0\par

```

Comment:
I think @melissa has already done some (or all) of this.
I will add a separate ticket.

The inclusion of the license files is giving some issues, so I'm not expecting this PR to close until Monday at least.